### PR TITLE
using correct path, should see labels on PR and be added to project b…

### PR DESCRIPTION
**Pull request by audreyandoy**
…oard

### CODE REVIEW

File: test_file.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Testing Labeler Action for general PR
\ No newline at end of file
+# Testing Labeler Action for general PR
+# okay using the correct path now.
```
Comment (audreyandoy) [inline test_file.md:2]: *add visual*

Comment (audrey-test) [inline test_file.md:2]: test

**Review by audrey-test — APPROVED** [review]

File: test_file.md (hunk 1 above)
Comment (audrey-test) [reply test_file.md:2]: test

**Review by audrey-test — CHANGES_REQUESTED** [review]

**Review by audrey-test — APPROVED** [review]

**Review by audreyandoy — COMMENTED** [review]
test

File: test_file.md (hunk 1 above)
Comment (audrey-test) [inline test_file.md:2]: approve changes

**Review by audrey-test — APPROVED** [review]

File: test_file.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# Testing Labeler Action for general PR
\ No newline at end of file
+# Testing Labeler Action for general PR
```
Comment (ada-instructor-1) [inline test_file.md:1]: test-approve comment

**Review by ada-instructor-1 — APPROVED** [review]

File: test_file.md (hunk 2 above)
Comment (ada-instructor-1) [inline test_file.md:1]: testing-approving-comment-pt-2

**Review by ada-instructor-1 — APPROVED** [review]

**Comment by audreyandoy** [comment]
test

**Comment by audreyandoy** [comment]
reopen to test review flow

File: test_file.md (hunk 2 above)
Comment (audrey-test) [inline test_file.md:1]: testing new review -approval

**Review by audrey-test — APPROVED** [review]

File: test_file.md (hunk 2 above)
Comment (ada-instructor-1) [inline test_file.md:1]: test second review approval... card should move to 'Magician Review'

**Review by ada-instructor-1 — APPROVED** [review]

**Pull request closed without merging**